### PR TITLE
add cloud-hypervisor

### PR DESCRIPTION
**Pull request by code-terror**
need approval for build

### CODE REVIEW

**Comment by code-terror** [comment]
need approval for build test and approval for merging cloud-hypervisor @jonathanmetzman
@DavidKorczynski @Navidem @oliverchang

File: projects/cloud-hypervisor/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Google LLC
```
Comment (Navidem) [inline projects/cloud-hypervisor/Dockerfile:1]: Wrong year.

File: projects/cloud-hypervisor/build.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
```
Comment (Navidem) [inline projects/cloud-hypervisor/build.sh:1]: Same here

**Review by Navidem — APPROVED** [review]
LGTM

**Pull request merged**